modules/op-resolver-http.adoc

```diff
@@ -0,0 +1,94 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="resolver-http-specify_{context}"]
+= Specifying a remote pipeline or task with the HTTP Resolver
+
+When creating a pipeline run or task run, you can specify a remote pipeline or task from an HTTP or HTTPS URL.
+
+.Prerequisites
+
+* You can optionally configure the service account for the HTTP resolver.
+
+.Procedure
+
+* To specify a remote pipeline or task from an HTTP URL, use the following reference format in the `pipelineRef` or `taskRef` spec:
++
+[source,yaml]
+----
+# ...
+  resolver: http
+  params:
+  - name: url
+    value: <fully_qualified_http_url>
+# ...
+----
++
+.Supported parameters for the HTTP Resolver
+|===
+| Parameter | Description | Example Value 
+
+| `url` 
+| The HTTP URL pointing to the Tekton resource to fetch. 
+| `https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/tasks/task-git-clone/0.4.0/task-git-clone.yaml` 
+|===
+
```

Comment:
This table should be relocated, either to the introduction of this procedure, or maybe to the top-level concept that precedes this procedure: https://78408--ocpdocs-pr.netlify.app/openshift-pipelines/latest/create/remote-pipelines-tasks-resolvers.html#resolver-http_remote-pipelines-tasks-resolvers

Since its a reference item, it doesn't belong in a Procedure. I can appreciate that its not really big enough to feel like it needs its own Reference module, but if you expect it to grow, that's also an option.